components/post_view/post_view.tsx

```diff
@@ -41,18 +54,21 @@ export default class PostView extends React.PureComponent {
         return null;
     }
 
-    changeUnreadChunkTimeStamp = (unreadChunkTimeStamp) => {
-        this.setState({
-            unreadChunkTimeStamp,
-            loaderForChangeOfPostsChunk: true,
-        }, () => {
-            window.requestAnimationFrame(() => {
-                this.setState({
-                    loaderForChangeOfPostsChunk: false,
+    changeUnreadChunkTimeStamp = (unreadChunkTimeStamp: number) => {
+        this.setState(
```

Comment:
Are the whitespace changes here required? Subjectively, the previous formatting seemed more familiar within this codebase.